HW0/P4/P4.py

```diff
@@ -40,37 +44,92 @@
     # Read the observation array and plot it (Part 2)
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    measured_inputs = np.loadtxt('P4_measurements.txt', delimiter=',')
+    stretch = np.array([[1.0/rx, 0.0, 0.0],[0.0, 1.0/ry, 0.0],[0.0, 0.0, 1.0/rz]])
+    stretched_inputs = np.dot(stretch, measured_inputs.T)
+    x_coords = stretched_inputs[0,:]
+    y_coords = stretched_inputs[1,:]
+    z_coords = stretched_inputs[2,:]
+    ax.plot(x_coords, y_coords, z_coords,
+            '.g', label='Observed trajectory')
 
     #####################
     # Part 3:
     # Use the initial conditions and propagation matrix for prediction
     #####################
 
-    # A = ?
-    # a = ?
-    # s = ?
-
     # Initial conditions for s0
-    # Compute the rest of sk using Eq (1)
+    A = np.array([[1.0, 0.0, 0.0, dt, 0.0, 0.0], [0.0, 1.0, 0.0, 0.0, dt, 0.0], [0.0, 0.0, 1.0, 0.0, 0.0, dt],
```

Comment:
This might be easier to read if you code on a new line after every comma, i.e. 

[
[0,0,0],
[0,0,0],
[0,0,0]
]

etc.